scripts/dag_imports.json

```diff
@@ -16,6 +16,10 @@
             "import": "from airflow.contrib.operators import bigquery_operator",
             "class": "bigquery_operator.BigQueryOperator"
         },
+        "BigQueryToBigQueryOperator": {
+            "import": "from airflow.contrib.operators import bigquery_to_bigquery",
```

Comment:
This note to our future selves is that importing from `airflow.providers` is planned in the roadmap and that's why I'm okay with this PR importing from `contrib` right now